### PR TITLE
= actor update java and scala doc to use the terminate method

### PR DESCRIPTION
**Pull request by He-Pin**
The method referenced in code's doc is not right.

### CODE REVIEW

**Review by chbatey — APPROVED** [review]
LGTM

File: akka-actor/src/main/scala/akka/actor/ActorSystem.scala [hunk 1]
```diff
@@ -502,26 +502,26 @@ abstract class ActorSystem extends ActorRefFactory {
...
   /**
-   * Register a block of code (callback) to run after ActorSystem.shutdown has been issued and
+   * Register a block of code (callback) to run after [[ActorSystem.terminate()]] has been issued and
    * all actors in this actor system have been stopped.
    * Multiple code blocks may be registered by calling this method multiple times.
    * The callbacks will be run sequentially in reverse order of registration, i.e.
    * last registration is run first.
    *
-   * Throws a RejectedExecutionException if the System has already shut down or if shutdown has been initiated.
+   * Throws a RejectedExecutionException if the System has already been terminated or if terminate has been initiated.
```
Comment (johanandren) [inline akka-actor/src/main/scala/akka/actor/ActorSystem.scala]: "or if termination has been initiated"

**Review by johanandren — CHANGES_REQUESTED** [review]

**Comment by hepin1989** [comment]
rebased

**Review by johanandren — APPROVED** [review]
LGTM, thanks @hepin1989 !

**Comment by hepin1989** [comment]
@johanandren If I found an issue , I will always try to fix it ,no matter how big or small it is :)

**Pull request merged**